app/2.0/docs/glossary.md

```diff
@@ -94,13 +94,18 @@ This tool has been replaced by [Polymer Analyzer](#polymer-analyzer). It is a st
 framework for Web Components. 
 
 ## L
+### lazy-imports
+Lazy Imports are a declarative way to describe the resources that you will import lazily, and
```

Comment:
This paragraph sounds a bit tautological to me :) I suggest rewording. Maybe something like this, edited for technical accuracy ;) :

"Lazy imports are a declarative way to describe the resources to be loaded lazily in an application (see #lazy-loading). Describing lazily-loaded resources declaratively means they can be linted, and that sharded bundling can be performed without configuration."

What is sharded bundling? We don't mention it in the glossary and idk what it means. I feel that we should define it in the glossary if we use it in another definition. 

Also not sure what it means to be able to do linting and sharded bundling without configuration. What config is needed if the resources are described imperatively? I am confuse :/

Not sure if others would share my confusion, however :)